src/lib/analytics/segmentOneTrustIntegration.ts

```diff
@@ -0,0 +1,209 @@
+import { data as sd } from "sharify"
+
+const SEGMENT_MAPPING = "C0001" // Segment itself assigned OneTrust Strictly Necessary category.
+const SEGMENT_WRITE_KEY = sd.SEGMENT_WRITE_KEY
+
+let PREVIOUS_CONSENT = ""
+let SEGMENT_DESTINATIONS = []
+let ONETRUST_ENABLED = false
+
+if (sd.ONETRUST_SCRIPT_ID) {
```

Comment:
@damassi - Wonder if this is sufficient, as over there in [src/v2/index.ejs](https://github.com/artsy/force/blob/0fdf31ad41419ac08562e2a979fb82b25d496531/src/v2/index.ejs#L7), the condition is more rigorous. I can copy the condition from there, but then it seems too much. wdyt?

Reply:
I'd consider whatever v2 is doing as canonical.

Reply:
got it. changed conditional to match v2 (except test for [!disable.onetrust flag](https://github.com/artsy/force/blob/eb1997b04759458aedcbe2cfc0369b9c5ae6fb8c/src/v2/index.ejs#L7) which doesn't work in the integration script.)